pkg/client/client.go

```diff
@@ -269,24 +269,26 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, allAddresses,
 	}
 
 	if sr.IPAM != nil {
-		var v4CIDR, v6CIDR string
-		v4AllocRange := localNode.PrimaryAddress.IPV4.AllocRange
-		v6AllocRange := localNode.PrimaryAddress.IPV6.AllocRange
+		var v4CIDR, v6CIDR, v4AllocRangeFmt, v6AllocRangeFmt string
 		if localNode != nil {
+			v4AllocRange := localNode.PrimaryAddress.IPV4.AllocRange
+			v6AllocRange := localNode.PrimaryAddress.IPV6.AllocRange
+			v4AllocRange = fmt.Sprintf(" allocated from %s", v4AllocRange)
+			v6AllocRange = fmt.Sprintf(" allocated from %s", v6AllocRange)
 			if nIPs := ip.CountIPsInCIDR(v4AllocRange); nIPs > 0 {
 				v4CIDR = fmt.Sprintf("/%d", nIPs)
 			}
 			if nIPs := ip.CountIPsInCIDR(v6AllocRange); nIPs > 0 {
 				v6CIDR = fmt.Sprintf("/%d", nIPs)
 			}
 		}
-		fmt.Fprintf(w, "IPv4 address pool:\t%d%s allocated from %s\n", len(sr.IPAM.IPV4), v4CIDR, v4AllocRange)
+		fmt.Fprintf(w, "IPv4 address pool:\t%d%s%s\n", len(sr.IPAM.IPV4), v4CIDR, v4AllocRangeFmt)
```

Comment:
`v4AllocRangeFmt` has no assigned value and it's an empty string. I don't think that was your intention. ;)

Also, do you know why `localNode` or any of its members are `nil`? And are you sure we can just ignore that? Correct me if I'm wrong, but I think that every node should have address allocation range, cilium-agent has default ranges which are used when user doesn't define its own ranges.

Reply:
> v4AllocRangeFmt has no assigned value

:man_facepalming: Thanks, fixed.

> Also, do you know why localNode or any of its members are nil? 

https://github.com/cilium/cilium/pull/6271 made setting of status fields asynchronous, so in the very beginning `localNode` can be still unset.

Reply:
OK, :man_facepalming: for me now, I even reviewed your PR about that. Looks good now, thanks!

Reply:
that was some slippery snippet 
![](https://github.githubassets.com/images/icons/emoji/unicode/1f926-2642.png)